src/coreclr/interpreter/compiler.cpp

```diff
@@ -2208,6 +2206,25 @@ static int32_t GetLdindForType(InterpType interpType)
     return -1;
 }
 
+static bool DoesValueTypeContainGCRefs(COMP_HANDLE compHnd, CORINFO_CLASS_HANDLE clsHnd)
+{
+    unsigned size = compHnd->getClassSize(clsHnd);
```

Comment:
Why not use `m_compHnd->getClassAttribs(clsHnd) & CORINFO_FLG_CONTAINS_GC_PTR` ?